package.json

```diff
@@ -48,7 +48,7 @@
     "form-data": "^3.0.0"
   },
   "peerDependencies": {
-    "graphql": "14.x || 15.x"
+    "graphql": "14.x || 16.x"
```

Comment:
Why drop 15?

Reply:
SMH, I wasn't even thinking. Clearly this should be a range.